server/sonar-server/src/main/java/org/sonar/server/issue/ws/SearchAction.java

```diff
@@ -176,20 +177,8 @@ public void define(WebService.NewController controller) {
     action.createParam(PARAM_TYPES)
       .setDescription("Comma-separated list of types.")
       .setSince("5.5")
-      .setPossibleValues(RuleType.values())
+      .setPossibleValues((Object[]) RuleType.values())
       .setExampleValue(format("%s,%s", RuleType.CODE_SMELL, RuleType.BUG));
-    action.createParam(DEPRECATED_PARAM_ACTION_PLANS)
```

Comment:
Removing these parameters should also remove its usage in IssueService, IssueIndex, IssuesWsParameters, SearchAction. No need to have facets on them